commit e9ddc08da0982f36581ae5a8c7763453ff41cfe8
Author: Sergey Poznyakoff <gray@gnu.org>
Date:   Thu Sep 25 00:22:16 2014 +0300

    Bugfixes.
    
    * doc/tar.1: Fix typo in font spec.
    * src/tar.c (sort_mode_arg, sort_mode_flag): Protect "inode"
    (SAVEDIR_SORT_INODE) with D_INO_IN_DIRENT

diff --git a/src/tar.c b/src/tar.c
index 225c624..f8102e0 100644
--- a/src/tar.c
+++ b/src/tar.c
@@ -1341,14 +1341,18 @@ static char filename_terminator;
 static char const *const sort_mode_arg[] = {
   "none",
   "name",
+#if D_INO_IN_DIRENT
   "inode",
+#endif
   NULL
 };
 
 static int sort_mode_flag[] = {
     SAVEDIR_SORT_NONE,
     SAVEDIR_SORT_NAME,
+#if D_INO_IN_DIRENT
     SAVEDIR_SORT_INODE
+#endif
 };
 
 ARGMATCH_VERIFY (sort_mode_arg, sort_mode_flag);